types/types.go

```diff
@@ -130,5 +136,7 @@ type Frontend interface {
 type DataProcessor interface {
 	ReaderWriterAt
 	PingResponse() error
+	Sync() error
+	Unmap(int64, uint32) error
 	//Update() error
```

Comment:
Since sync and unmap is used in many packages like types , replica , frontend etc. Can we write reason what is the need of this in every package, since it will be very difficult for the new comer to understand why the same function is being used everywhere ?